po/tr.po

```diff
@@ -0,0 +1,706 @@
+
+#: src/download.rs:321
+msgid " PKGBUILDs up to date"
+msgstr " PLGBUILD'leri güncel"
+
+#: src/download.rs:520
+msgid "{} is not installed: can not get repo packages"
+msgstr "{} yüklü değil: depo paketleri alınamadı"
+
+#: src/download.rs:561
+msgid "asp returned {}"
+msgstr "asp cevabı {}"
+
+#: src/exec.rs:86
+msgid "Pacman is currently in use, please wait..."
+msgstr "Pacman şuan kullanılıyor, lütfen bekleyiniz..."
+
+#: src/fmt.rs:6 src/info.rs:146 src/search.rs:176
+msgid "None"
+msgstr "Hiç biri"
+
+#: src/info.rs:30 src/query.rs:51
+msgid "package '{}' was not found"
+msgstr "'{}' paketi bulunamadı"
+
+#: src/info.rs:57 src/info.rs:96
+msgid "Name"
+msgstr "İsim"
+
+#: src/info.rs:58 src/info.rs:97
+msgid "Version"
+msgstr "Versiyon"
+
+#: src/info.rs:59 src/info.rs:98
+msgid "Description"
+msgstr "Açıklama"
+
+#: src/info.rs:60 src/info.rs:106
+msgid "Groups"
+msgstr "Gruplar"
+
+#: src/info.rs:61 src/info.rs:107
+msgid "Licenses"
+msgstr "Lisanslar"
+
+#: src/info.rs:62 src/info.rs:108
+msgid "Provides"
+msgstr "Sağladıkları"
+
+#: src/info.rs:63 src/info.rs:109
+msgid "Depends On"
+msgstr "Gereksinimleri"
+
+#: src/info.rs:64 src/info.rs:110
+msgid "Make Deps"
+msgstr "Make Gereksinimleri"
+
+#: src/info.rs:65 src/info.rs:111
+msgid "Check Deps"
+msgstr "Gereksinimleri Kontrol Et"
+
+#: src/info.rs:66 src/info.rs:113
+msgid "Conflicts With"
+msgstr "Çakıştıkları"
+
+#: src/info.rs:67 src/info.rs:114
+msgid "Maintainer"
+msgstr "Bakıcı"
+
+#: src/info.rs:68 src/info.rs:115
+msgid "Votes"
+msgstr "Oylar"
+
+#: src/info.rs:69 src/info.rs:116
+msgid "Popularity"
+msgstr "Popülerite"
+
+#: src/info.rs:70 src/info.rs:117
+msgid "First Submitted"
+msgstr "İlk Gönderim"
+
+#: src/info.rs:71 src/info.rs:118
+msgid "Last Modified"
+msgstr "Son Değişiklik"
+
+#: src/info.rs:72 src/info.rs:120
+msgid "Out Of Date"
+msgstr "Tarihi Geçmiş"
+
+#: src/info.rs:73 src/info.rs:125
+msgid "ID"
+msgstr "ID"
+
+#: src/info.rs:74 src/info.rs:126
+msgid "Package Base ID"
+msgstr "Baz Paket ID"
+
+#: src/info.rs:75 src/info.rs:127
+msgid "Keywords"
+msgstr "Anahtar Kelimeler"
+
+#: src/info.rs:76 src/info.rs:129
+msgid "Snapshot URL"
+msgstr "Snapshot URL"
+
+#: src/info.rs:92
+msgid "No"
+msgstr "Hayır"
+
+#: src/info.rs:112
+msgid "Optional Deps"
+msgstr "Opsiyonel Gereksinimler"
+
+#: src/install.rs:131 src/install.rs:545 src/install.rs:944
+msgid "no architecture"
+msgstr "mimari yok"
+
+#: src/install.rs:138
+msgid "can't build package as root"
+msgstr "root olarak paket kurulamadı"
+
+#: src/install.rs:142
+msgid "failed to parse srcinfo generated by makepkg"
+msgstr "makepkg tarafından oluşturulmuş srcinfo çekilemedi"
+
+#: src/install.rs:171 src/install.rs:442
+msgid "Resolving dependencies..."
+msgstr "Gereksinimler çözülüyor..."
+
+#: src/install.rs:220 src/install.rs:225
+msgid "failed to download sources"
+msgstr "kaynaklar indirilemedi"
+
+#: src/install.rs:230 src/install.rs:246 src/install.rs:254
+msgid "failed to build"
+msgstr "kurulamadı"
```

Comment:
"`kurulamadı`" == "failed to install"

You might want to change the wording here. (and other places as well)